completions/shenv.bash

```diff
@@ -6,8 +6,8 @@ _shenv() {
     COMPREPLY=( $(compgen -W "$(shenv commands)" -- "$word") )
   else
     local words=("${COMP_WORDS[@]}")
-    unset words[0]
```

Comment:
E.g. this would fail if not quoted and there's a file `words0` in the current dir.